examples/dll-app-and-vendor/0-vendor/README.md

```diff
@@ -1,10 +1,10 @@
 This is the vendor build part.
 
-It's built separately from the app part. The vendors dll is only built when vendors has changed and not while the normal development cycle.
+It's built separately from the app part. The vendors dll is only built when the array of vendors has changed and not during the normal development cycle.
```

Comment:
Update the `template.md` file too.

README.md is generated from this file.

Reply:
@sokra I don't see the text here to make that change: https://github.com/stepharr/webpack/edit/master/examples/dll-app-and-vendor/1-app/template.md.

Reply:
@stepharr He means this one https://github.com/stepharr/webpack/blob/master/examples/dll-app-and-vendor/0-vendor/template.md